CNV-35207_1: [Docs] Cloning from cached snapshots in UI (#71208)
openshift/openshift-docs: modules/virt-cloning-vm-snapshot.adoc

// Module included in the following assemblies:
//
// * virt/backup_restore/virt-backup-restore-snapshots.adoc
:_mod-docs-content-type: PROCEDURE
[id="virt-cloning-vm-snapshot_{context}"]

> = Cloning a snapshot to create a VM by using the web console

Reviewer: Snapshot can't be cloned
VM can be cloned

Author: @ShellyKa13  and @jpeimer, I've renamed the topic/procedure to *Cloning a VM by using the web console* and updated the steps. 

Just as an aside, when I click *Clone* in the last step, I get an error message saying "An error occurred Error "Virtual Machine volume rootdisk does not support snapshots" for field "spec.Source"."

> You can clone an existing virtual machine (VM) snapshot to create a new VM.

Reviewer: You can clone an existing virtual machine (VM) to a new VM, if VM's underlying storage supports snapshots.

.Procedure

> . Navigate to *Virtualization* -> *VirtualMachines* in the web console.
> . Select a VM to open the *VirtualMachine details* page.
> . Click the *Snapshots* tab.
> . Click the options menu {kebab} for the snapshot you want to clone and select *Clone*.
> . On the *Clone VirtualMachineSnapshot* page, enter the name of the new VM.
> . (Optional) Select the *Clone* checkbox to start the cloned VM.

Reviewer: ```suggestion
. Navigate to *Virtualization* -> *VirtualMachines* in the web console.
. Select a VM to open the *VirtualMachine details* page.
. Click the *Actions* in the right top -> *Clone*
. (Optional) Select the *Clone* checkbox to start the cloned VM.
```

// Module included in the following assemblies:
//
// * virt/backup_restore/virt-backup-restore-snapshots.adoc
:_mod-docs-content-type: PROCEDURE
[id="virt-cloning-vm-snapshot_{context}"]
= Cloning a VM by using the web console

> You can create a new VM by cloning either an existing VM or a VM snapshot.

Reviewer 2: I think the sentence should be changed a bit to something like
`You can create a new VM either by cloning an existing vm or using a VM snapshot as a source`
since with current phrasing the cloning verb is still on the vm snapshot

.Procedure

> . Navigate to *Virtualization* -> *VirtualMachines* in the web console.

Reviewer 2: So this procedure explains only the part of cloning an existing VM.
Since you mentioned that `You can create a new VM by cloning either an existing VM or a VM snapshot.` It seems like you also need to explain how to do it from a a vmsnapshot source which is from what I understand is done by going to the snapshot tab and have an action on the snapshot of create vm.

Reviewer: I see that it is now explained in modules/virt-creating-vm-from-snapshot-web.adoc
So should be good?

// Module included in the following assemblies:
//
// * virt/virtual_machines/creating_vms_rh/creating-vms-from-instance-types.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-cloning-vm-snapshot_{context}"]
= Cloning a VM by using the web console

You can clone an existing VM by using the web console.

.Procedure

> . Navigate to *Virtualization* -> *VirtualMachines* in the web console.

Reviewer 3: ```suggestion
. Go to *Virtualization* -> *VirtualMachines* in the web console.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n

Author: Same comment.

. Click the *Snapshots* tab.
. Click the options menu {kebab} for the snapshot you want to clone and select *Clone*.

> . On the *Clone VirtualMachineSnapshot* page, enter the snapshot name.

Reviewer 4: Here it seems like we are creating a new snapshot based on the original one. Instead, we are creating a VM.  In the step here we have `enter the snapshot name.` but instead we are entering the new VM name

Author: This procedure has changed and is now **Cloning a VM by using the web console**, with updated steps
